packages/contracts-bedrock/scripts/deploy/AddGameType.s.sol

```diff
@@ -62,6 +68,23 @@ contract AddGameType is Script {
         vm.store(prank, bytes32(0), bytes32(uint256(uint160(address(_agi.opcmImpl)))));
         vm.label(prank, "DummyCaller");
 
+        if (useV2) {
+            // V2 path: Use upgrade() with updated game configs
+            return addGameTypeWithV2(_agi, prank);
```

Comment:
note to reviewers: `prank` is a bad name here, this var actually represents the ProxyAdminOwner.